frameworks/cassandra/tests/config.py

```diff
@@ -1,17 +1,21 @@
 """ Utilties specific to Cassandra tests """
 
 import os
+import logging
+import traceback
 
 import sdk_hosts
 import sdk_jobs
 import sdk_plan
 import sdk_utils
 
+log = logging.getLogger(__name__)
 
 PACKAGE_NAME = 'cassandra'
 DEFAULT_TASK_COUNT = 3
 FOLDERED_SERVICE_NAME = sdk_utils.get_foldered_name(PACKAGE_NAME)
 DEFAULT_CASSANDRA_TIMEOUT = 600
+SERVICE_NAME = os.environ.get('SERVICE_NAME') or PACKAGE_NAME
```

Comment:
1. Seems like this doesn't actually get used
2. @mellenburg is working on standardizing the consistency across service names and IIRC we're going with `FOLDERED_SERVICE_NAME` so we should probably use that everywhere